examples/api/lib/material/theme_data/theme_data.0.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Template: dev/snippets/config/templates/stateless_widget_scaffold.tmpl
+//
+// Comment lines marked with "▼▼▼" and "▲▲▲" are used for authoring
+// of samples, and may be ignored if you are just exploring the sample.
+
+// Flutter code sample for ThemeData
+//
+//***************************************************************************
+//* ▼▼▼▼▼▼▼▼ description ▼▼▼▼▼▼▼▼ (do not modify or remove section marker)
+
+// This sample shows how to display a [Container] widget that uses [Theme.of]
+// to retrieve the primary color from the theme's [colorScheme] to draw an
+// amber square.
+// The [Builder] widget separates the parent theme's [BuildContext] from the
+// child's [BuildContext].
+
+//* ▲▲▲▲▲▲▲▲ description ▲▲▲▲▲▲▲▲ (do not modify or remove section marker)
+//***************************************************************************
```

Comment:
These lines can all be replaced with: 
```suggestion
// Flutter code sample for ThemeData
```

Reply:
Does it still work automatically then?

Reply:
All of the examples have been moved out of the API docs, and they're not going to be re-inserted, so there's no need for all of the sections and markup.

See the [README.md](https://github.com/flutter/flutter/blob/master/examples/api/README.md#authoring) for more info.

Reply:
@gspencergoog Which part of the README are you referring to? I only see the "** See code in " part.

Reply:
Just the part about authoring examples. 

The answer to your question is that we're not parsing these examples anymore, we're just including them as-in the API docs, so you don't need all the comment markers anymore.